README.md

```diff
@@ -65,6 +65,7 @@ Options:
   -o, --only                   Only build specified module, or comma separated
                                list of modules. All others are ignored.
   -b, --debug                  Build debug version of modules
+  -tp, --tag-prefix            github tag prefix, default is "v"
```

Comment:
Can this be `--prebuild-tag-prefix` to be clear this is just for prebuild stuff

Reply:
Sure thing. Will update.